addons/mail/static/src/models/file_uploader.js

```diff
@@ -128,9 +128,13 @@ registerModel({
                     return;
                 }
                 try {
+                    const body = this._createFormData({ composer, file, thread });
+                    if (activity && activity.exists()) {
```

Comment:
you told me on discord that the exists wasn't needed no ?

Reply:
Agree that `.exists()` looks unnecessary. This is needed only when the record may be deleted, which happens mostly after an asynchronous task or when there's an update on records that may have side-effect to delete the record in a prior synchronous statement. None of them are present here, so only falsy value on accessing the relational value above should be handled, not `.exists()`.

Reply:
Indeed sorry, thought that I already pushed without the `exists`, done, thanks!